lib/scholar/integrate/integrate.ex

```diff
@@ -49,13 +104,529 @@ defmodule Scholar.Integrate do
   deftransform integrate_uniform(y, method, opts \\ []) do
     case method do
       :trapezoidal ->
-        Scholar.Integrate.Trapezoidal.trapezoidal_uniform(y, opts)
+        trapezoidal_uniform(y, opts)
 
       :simpson ->
-        Scholar.Integrate.Simpson.simpson_uniform(y, opts)
+        simpson_uniform(y, opts)
 
       _ ->
         raise ArgumentError, "Unknown integration method: #{inspect(method)}"
     end
   end
```

Comment:
If we are having the functions in this module, I no longer think we need integrate, because you can get the same behaviour by doing:

    apply(Scholar.Integrate, :trapezoidal, [y, x])

Thoughts?

Reply:
Yes, valid point. Now, `Scholar.Integrate.integrate/integrate_uniform` is a little bit redundant.

Reply:
Same issue I have with pipes. Needing apply will imply needing then and captures.

IMO we should have only integrate and integrate_uniform

Reply:
@polvalente I am really not in favor of arguing for the existence of functions for "pipe" reasons. A function needs to exist by itself. It is very easy to wrap apply into something pipeable.

Reply:
pipe usability IMO should be considered because it is one major use case of these functions. Needing apply will force people away from piping into these functions